src/Pelagos/Entity/ResearchGroup.php

```diff
@@ -652,13 +661,39 @@ public function setEmailAddress($emailAddress)
      *
      * @access public
      *
-     * @return string Containing emailADdress.
+     * @return string Containing emailAddress.
      */
     public function getEmailAddress()
     {
         return $this->emailAddress;
     }
 
+    /**
+     * Setter for lockStatus.
+     *
+     * @param boolean $state Containing desired state of lock.
+     *
+     * @access public
+     *
+     * @return void
+     */
+    public function setLockStatus(bool $state)
```

Comment:
Maybe a bit odd, but I like the functions: lock, unlock and isLocked. That way we have a bit more control. For instance when unlocked making sure it's unlockable, and not already unlocked, or something in the future that prevents locking.